runtime/tests/interpreter/builtinfunctions_test.go

```diff
@@ -147,6 +149,53 @@ func TestInterpretToBytes(t *testing.T) {
 	})
 }
 
+func TestInterpretAddressFromBytes(t *testing.T) {
+
+	t.Parallel()
+
+	runAddressTest := func(t *testing.T, expected []byte, innerCode string) {
+		t.Run(innerCode, func(t *testing.T) {
+			t.Parallel()
+
+			code := fmt.Sprintf("fun test(): Address { \n return Address.fromBytes(%s) \n }", innerCode)
+
+			inter := parseCheckAndInterpret(t, code)
+			res, err := inter.Invoke("test")
+
+			require.NoError(t, err)
+
+			addressVal, ok := res.(interpreter.AddressValue)
+			require.True(t, ok)
+
+			require.Equal(t, expected, addressVal.ToAddress().Bytes())
+		})
+	}
+
+	runAddressTest(t, []byte{1}, "[1]")
+	runAddressTest(t, []byte{12, 34, 56}, "[12, 34, 56]")
+	runAddressTest(t, []byte{67, 97, 100, 101, 110, 99, 101, 33}, "[67, 97, 100, 101, 110, 99, 101, 33]")
+}
+
+func TestInterpretAddressFromBytesInvalid(t *testing.T) {
```

Comment:
Maybe group this and the test above into one test function with multiple subtests (`t.Run`).

Also, could you please add a test case which checks that `Address.fromButes(someAddress.toBytes()) == someAddress` holds for various `someAddress`?

Reply:
Added these cases.